src/main/java/org/craftercms/studio/api/v2/annotation/RetryingRepositoryOperationAspect.java

```diff
@@ -69,26 +77,35 @@ public Object doRetryingOperation(ProceedingJoinPoint pjp) throws Throwable {
         do {
             numAttempts++;
             try {
-				 // Execute the business code again
+                // Execute the business code again
                 if (numAttempts > 1) {
-                    logger.debug("Retrying repository operation attempt " + (numAttempts - 1));
+                    logger.debug("Retrying repository operation attempt {}", numAttempts - 1);
                 }
                 return pjp.proceed();
             } catch (JGitInternalException | GitCliException ex) {
                 lastException = ex;
                 if (isRepositoryLocked(ex)) {
-                    logger.debug("Failed to execute " + method.getName() + " after " + numAttempts + " attempts", ex);
-                    if (numAttempts < maxRetries) {
-                        // If the maximum number of retries is not reached, sleep and execute it again
-                        long sleep = (long) (Math.random() * maxSleep);
-                        logger.debug("Git operation failed due to the repository being locked. Will wait for " +
-                                     sleep + " before next retry" + method.getName());
-                        Thread.sleep(sleep);
+                    logger.debug("Failed to execute {} after {} attempts", method.getName(), numAttempts, ex);
```

Comment:
Please use '{}' for placeholders instead of just {}

Reply:
Fixed. Thank you!